### PR TITLE
added drone cli

### PR DESCRIPTION
**Pull request by eyenx**
drone cli from drone.io / using env variables DRONE_TOKEN & DRONE_SERVER

### CODE REVIEW

**Comment by eyenx** [comment]
@bfirsh Is the project still alive?

**Comment by tjamet** [comment]
@eyenx Yes, I am contributing as maintainer so far, though I didn't go on the packages part yet
Will do it sometime soon

**Comment by eyenx** [comment]
I would love to maintain too. I just am not that fresh with GoLang, but would be perfect to learn, right?

**Comment by tjamet** [comment]
@eyenx Please do submit patches you think are useful for the project. I am trying to get as responsive as possible regarding contributions and feedback :)

**Comment by eyenx** [comment]
@tjamet any updates?

File: drone/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM drone/cli
+LABEL io.whalebrew.config.environment '["DRONE_SERVER"]'
+LABEL io.whalebrew.config.environment '["DRONE_TOKEN"]'
```
Comment (tjamet) [inline drone/Dockerfile:3]: Unless I am mistaking, labels are not extended but overridden, to get both environment variables you need to do this:

```suggestion
LABEL io.whalebrew.config.environment '["DRONE_SERVER", "DRONE_TOKEN"]'
```

**Comment by tjamet** [comment]
can you also add the package to the list of packages to build as described in #65 ?

**Comment by eyenx** [comment]
@tjamet done

**Pull request merged**